packages/opera.rb

```diff
@@ -28,5 +28,7 @@ def self.install
     FileUtils.mv 'usr/bin', "#{CREW_DEST_PREFIX}"
     FileUtils.mv 'usr/lib', "#{CREW_DEST_PREFIX}"
     FileUtils.mv 'usr/share', "#{CREW_DEST_PREFIX}"
+    system "echo 'if [[ $(ls #{CREW_PREFIX}/bin/ | grep x-www-browser) = x-www-browser ]]; then echo x-www-browser is already set; else ln -s #{CREW_PREFIX}/bin/opera #{CREW_PREFIX}/bin/x-www-browser; fi' > script.la.tmp"
+    system "bash script.la.tmp"
   end
 end
```

Comment:
Please remove the opera update from this PR.

Reply:
Done